.env.template

```diff
@@ -31,5 +31,8 @@ AZURE_DOCUMENT_INTELLIGENCE_ADMIN_KEY=
 AZURE_LANGUAGE_SERVICE_ENDPOINT=
 AZURE_LANGUAGE_SERVICE_KEY=
 
+#### Multithreading
+MAX_WORKER_THREADS= #Leave empty if you want it to be optimized by cpu cores
```

Comment:
If most people will leave this empty, I suggest we comment this out entirely (like the logging stuff), and only ask people to uncomment, and set a value, if they want to be specific about worker threads

Reply:
I agree with you, fixed